src/core/contracts/identity.test.ts

```diff
@@ -166,6 +171,119 @@ describe("identity", () => {
       expect(await isAuthorizedTo(NON_OWNER, contractAddress, 1, 60)).toBeTruthy();
     });
   });
+
+  describe("#getDomainSeperator", () => {
```

Comment:
Separator (it should have two "a"s)  is misspelled here and in the function. Normally I would not make spelling errors a blocking request, but people won't be able to find it if they search on the correct spelling.

Reply:
🤦